.gitignore

```diff
@@ -1 +1,2 @@
 /build
+/CMakeLists.txt.user
```

Comment:
Why add this file to the gitignore?

Reply:
CMakeLists.txt.user is the result of your IDE based on packages requested by your CMake file. It is very specific for the machine this code is compiled on. Why drag this from clone to clone and have it confuse the IDE of the one cloning this. It can be the cause of a lot of unlogical errors.